### ISSUE TITLE
Add Claimer Role

### ISSUE THREAD

**Issue by Jordan-Cottle** [Jordan-Cottle/screeps-kotlin#9]
For claiming new rooms

**Comment by Jordan-Cottle** [comment]
For now, probably want this to be manually triggered or created.

Probably worth having some kind of "nextRoom" in memory that can be populated manually. When "nextRoom" is not already owned by us and our global controller level allows for expansion, the closest spawn capable of spawning a creep should create the claimer and send it off.

**Comment by Jordan-Cottle** [comment]
Depends on #10 

Having an extra room is useless if it won't do anything.
